crates/eframe/src/web/web_painter_wgpu.rs

```diff
@@ -59,14 +77,20 @@ impl WebPainterWgpu {
     pub async fn new(canvas_id: &str, options: &WebOptions) -> Result<Self, String> {
         log::debug!("Creating wgpu painter");
 
+        // Workaround for https://github.com/gfx-rs/wgpu/issues/3710:
+        // Don't use `create_surface_from_canvas`, but `create_surface` instead!
```

Comment:
You could keep the old code behind `if false { … }` so we can more easily switch back when the bug is fixed